README.md

```diff
@@ -58,8 +58,19 @@ See [Amplication website](http://amplication.com/) or [Amplication docs](http://
 
 ## Development
 
-Amplication is constructed from multiple packages. To initialize the local development environment, run `npm run setup:dev`, it will install all the required dependencies, run the necessary scripts and init Docker-based Postgres server.
-To set up manually Amplication local development environment, follow the "Amplication Server" and "Amplication Client" installation instructions.
+Amplication is using a mono-repo with multiple packages. To initialize all the packages on a local development environment, follow these steps:
+
+First, execute this command, in the root folder of the projec:
+```
+npm install
+```
+Then, execute this command, also in the root folder of the project:
+```
+npm run setup:dev
+```
+The second command will install all the required dependencies, run the necessary scripts and init a Docker-based Postgres server.
```

Comment:
This will install all the required dependencies, run the necessary scripts and init a Docker-based Postgres server.

Reply:
```suggestion

This will install all the required dependencies, run the necessary scripts and init a Docker-based Postgres server.
```